package.json

```diff
@@ -68,7 +73,10 @@
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-react": "^7.24.0",
-    "jest": "^27.0.6"
+    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-unused-imports": "^1.1.2",
+    "jest": "^27.0.6",
+    "typescript": "~4.0.0"
```

Comment:
@filiptronicek Why `~` instead of `^` 🤔 

https://stackoverflow.com/a/22345808/8860100